src/pyupgrade/plugins/format_specififiers.rs

```diff
@@ -15,7 +15,7 @@ use crate::cst::matchers::{match_call, match_expression};
 // The regex documentation says to do this because creating regexs is expensive:
 // https://docs.rs/regex/latest/regex/#example-avoid-compiling-the-same-regex-in-a-loop
 lazy_static! {
```

Comment:
We use `Lazy` for this, you can grep for some other usages of `Regex` to see an example :)